### PR TITLE
Added unit testing for mentor service file

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
**Description**

Added unit testing for mentor service file
Test suites for getting all mentors, finding mentor by id and updating a mentor

**Type of Change:**
- [x] Testing

**How Has This Been Tested?**

Terminal commands to test code:-
1). npm run lint
2). npm run lint:fix
3). nest start
4). npm run test

**Screenshots**

image

### CODE REVIEW

File: src/modules/mentor/mentor.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      jest.spyOn(model, 'findById').mockReturnValueOnce(
+        createMock<Query<MentorDoc, MentorDoc>>({
+          exec: jest.fn().mockResolvedValueOnce(mockMentorDoc()),
+        }),
+      );
+      const findMockMentor = mockMentor();
+      const id = new mongoose.Schema.Types.ObjectId('22', 0, 'rtex');
+      const foundMentor = await service.findMentorById(id);
+      expect(foundMentor).toEqual(findMockMentor);
+    });
```
Comment (Abhishek-kumar09) [inline src/modules/mentor/mentor.service.spec.ts:113]: Why this test is skipped?

Comment (Devesh21700Kumar) [reply src/modules/mentor/mentor.service.spec.ts]: This is fixed now too

Comment (Devesh21700Kumar) [reply src/modules/mentor/mentor.service.spec.ts]: @Abhishek-kumar09  fixed

**Comment by Abhishek-kumar09** [comment]
I see there are two skipped tests in different models too, please fix those.

**Comment by Devesh21700Kumar** [comment]
> I see there are two skipped tests in different models too, please fix those.

Sure. Will fix that in a different PR. Mentor endpoints have no skipped tests now.

**Comment by Devesh21700Kumar** [comment]
> I see there are two skipped tests in different models too, please fix those.

@Abhishek-kumar09 both fixed now

**Review by Devesh21700Kumar — COMMENTED** [review]
image


@Abhishek-kumar09  all tests pass now and none are skipped

**Pull request merged**